SI/Sem.03/Pract.03/Task 04 - 06/Task04.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    unsigned int m, n, k = 0;
```

Comment:
Good solutions with handling alignment of 2-digit numbers.
However, if we exceed them, we will need to adjust alignment again(3-digits etc).
Kudos tho :clap: